Application/Features/BookingFeature/BookingProfile.cs

```diff
@@ -12,12 +12,27 @@ public BookingProfile()
             CreateMap<Booking, BookingDto>()
                 .ForMember(dest => dest.UserName, opt => opt.MapFrom(src => src.User.UserName));
 
+            // Is it ok to return all information thi way? or should we query the database separately for each required information?
```

Comment:
It depends on agreement.
I am 90% sure EF core will query with several call to db, at least you can look at profiler